pkg/telemetry/telemetry.go

```diff
@@ -271,6 +281,24 @@ func computeUsageMetrics(kubeletMetrics map[string]*io_prometheus_client.MetricF
 	return resourceMetrics, aggregateMetricValues(kubeletCPUSeconds.Metric), nil
 }
 
+func computeMicroShiftVersionMetric() (Metric, error) {
+	osVersion, err := getOSVersion()
+	if err != nil {
+		return Metric{}, fmt.Errorf("failed to get OS version: %v", err)
+	}
+	currentTimestamp := time.Now().UnixNano() / time.Millisecond.Nanoseconds()
+	return Metric{
+		Name: MetricNameMicroShiftVersion,
+		Labels: []MetricLabel{
+			{Name: LabelNameVersion, Value: prerun.GetVersionStringOfData()},
```

Comment:
Is this the right function call? `GetVersionStringOfData()` returns `string(version)` contained in `/var/lib/microshift/version`. 
Shouldn't we use the version of the executable?